src/mahoji/commands/smelt.ts

```diff
@@ -120,8 +117,25 @@ export const smeltingCommand: OSBMahojiCommand = {
 				timeToSmithSingleBar *= 1.075;
 				boosts.push('-7.5% penalty for not having graceful equipped.');
 			}
+			if (user.hasItemEquippedAnywhere('Smithing master cape')) {
+				timeToSmithSingleBar /= 2;
+				boosts.push('2x boost for Smithing master cape');
+			}
+			if (user.hasItemEquippedAnywhere(itemID('Dwarven gauntlets')) && bar.id !== itemID('Gold bar')) {
+				boosts.push('2x boost for having a Dwarven gauntlets equipped.');
+				timeToSmithSingleBar /= 2;
+			}
 		}
 
+		if (!blast_furnace) {
+			if (user.hasItemEquippedAnywhere(itemID('Dwarven gauntlets'))) {
```

Comment:
no need for itemID here (And elsewhere in this pr)

Reply:
Removed itemID from this PR

Reply:
Think I misinterpreted this, so added itemID back in some situations, let me know if you need it removed elsewhere